### PR TITLE
110 add zip format info link

### PR DESCRIPTION
**Pull request by Thykof**
image

### CODE REVIEW

**Review by Trottoir — APPROVED** [review]

File: resource/html/front/website.html [hunk 1]
```diff
@@ -119,10 +119,12 @@ <h4 class="dns-error">Website Name must be only lowercase letters and numbers</h
...
 					<h4 class="website-card-label">Website File</h4>
-					<button class="primary-button" id="file-select-button">
-						Import From
-
-					</button>
+					<div>
+						<button class="primary-button" id="file-select-button">
+							Import From
+						</button>
+						<p><a href="https://github.com/massalabs/thyra#upload-a-webstite" target="_blank">Format info</a></p>
```
Comment (SebastienValla) [inline resource/html/front/website.html:126]: same thing add rel="noopener" with target_blank for safety issue

**Review by SebastienValla — CHANGES_REQUESTED** [review]

**Review by SebastienValla — APPROVED** [review]

File: pkg/front/website/html.go [hunk 2]
```diff
@@ -126,10 +126,12 @@ const HTML = `<!DOCTYPE html>
...
-					<button class="primary-button" id="file-select-button">
-						Import From
-
-					</button>
+					<div>
+						<button class="primary-button" id="file-select-button">
+							Import From
+						</button>
+						<p><a href="https://github.com/massalabs/thyra#upload-a-webstite" target="_blank">Format info</a></p>
+					</div>
```
[the remark this answers is not in the document]
Comment (Thykof) [reply pkg/front/website/html.go:134]: as we talk together, rel="noopener" is by default when target is _blank (https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener)

**Comment by SlnPons** [comment]
@Thykof 2 comments: 
1. could you make the text smaller and grey please? (like the text info for the name above)
2. When I click on the link, I'm redirected to the readme but I can't see any section regarding the website upload. Are you sure you push the modification? Happy to help if needed

**Comment by Thykof** [comment]
@SlnPons I have just pushed the work we did together. Juste to keep track: about the link to the readme, it will work when this PR will be merge into main branch.

**Review by SlnPons — APPROVED** [review]

File: resource/html/front/wallet.html [hunk 3]
```diff
@@ -53,24 +53,24 @@ <h2>Thyra</h2>
...
 					<div class="row">
 						<div id="create-form">
-							<h2 class="mb-4">Create Wallet</h2>
+							<h2 class="mb-4">Create a wallet</h2>
 
 							<div class="form-floating mb-3 col-md-10">
 								<input
 									class="form-control"
-									placeholder="Nickname"
+									placeholder="Wallte name"
```
[the remark this answers is not in the document]
Comment (Thykof) [reply resource/html/front/wallet.html]: good catch, fixed

**Pull request merged**